src/orion/storage/base.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# pylint:disable=protected-access,too-many-public-methods
```

Comment:
Where is the protected access? Can you move `too-many-public-methods` to BaseStorageProtocol.